samples/settings/boards/esp32.conf

```diff
@@ -0,0 +1,38 @@
+CONFIG_WIFI=y
+CONFIG_WIFI_ESP32=y
+#CONFIG_HEAP_MEM_POOL_SIZE=131072
+CONFIG_HEAP_MEM_POOL_SIZE=98304
+
+CONFIG_NET_L2_ETHERNET=y
+
+CONFIG_NET_DHCPV4=y
+
+CONFIG_NET_CONFIG_LOG_LEVEL_DBG=y
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_NEED_IPV4=y
+
+CONFIG_MBEDTLS_ENTROPY_ENABLED=y
+CONFIG_MBEDTLS_KEY_EXCHANGE_ECDHE_ECDSA_ENABLED=y
+CONFIG_MBEDTLS_ECP_ALL_ENABLED=y
+
+
+#CONFIG_NET_L2_WIFI_SHELL=y
+#CONFIG_GOLIOTH_SYSTEM_SETTINGS=y
+#CONFIG_GOLIOTH_SAMPLE_WIFI_SETTINGS=y
+CONFIG_ESP32_WIFI_STA_AUTO=y
+CONFIG_GOLIOTH_SAMPLE_WIFI=n
+# CONFIG_GOLIOTH_SAMPLE_WIFI_SSID="Fim do Mundo"
+# CONFIG_GOLIOTH_SAMPLE_WIFI_PSK="19911996"
+
+CONFIG_MCUMGR=n
+CONFIG_MCUMGR_SMP_SHELL=n
+CONFIG_MCUMGR_CMD_CFG_MGMT=n
+CONFIG_MCUMGR_CMD_OS_MGMT=n
+
+CONFIG_GOLIOTH_SYSTEM_SERVER_HOST="coap.golioth.net"
+
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_NVS=y
+
+CONFIG_HWINFO=y
```

Comment:
Why do we need this? We don't use that in other samples.

Reply:
@mniestroj  I added the `CONFIG_HWINFO` to be able to check the device serial number or unique ID.I think is useful to add that to the other boards on the setting sample, so users can get a real hardware ID to be used to register the device on our platform.

Reply:
I think it's a good idea to extract the hardware ID from the device and register it on the cloud, rather than generating one in the cloud.

Reply:
i kept it as is, right?